views/admin.handlebars

```diff
@@ -31,4 +31,9 @@
   <input class="form-control" name="name" placeholder="Name" />
   <button class="btn btn-primary" type="submit"><i class="fas fa-plus"></i> Add</button>
 </form>
-
+<div class="download" style="margin-top: 50px">
+  <h3>Download Zapp usage data</h3>
+  <form action="/admin/download/Zapp_usage_data.csv" method="GET">

```

Comment:
This does not need to be a form, it can just be an `a` tag link with the `btn btn-primary` classes. See the download button on the home page.